### ISSUE TITLE
When fine-tuning is it necessary to set flow shift in training noise scheduler?

### ISSUE THREAD

**Issue by xilanhua12138** [Wan-Video/Wan2.1#291]
I noticed that in technical report, it mentions that in training, you use the logit-normal to sample time steps, do you additionally add flow shift to training noise scheduler?

**Comment by xilanhua12138** [comment]
@wan-x-ai

**Comment by wan-x-ai** [comment]
Yes, we adjust flow_shift based on resolution. For 480p video training, set flow_shift to 2. For higher resolutions (e.g., 720p or above), increase flow_shift appropriately to ensure optimal performance.

**Comment by xilanhua12138** [comment]
So it is logit normal + flow shift, the major time steps will be sampled at very big number?  Like this

![Image](https://github.com/user-attachments/assets/50343457-ad60-4cf1-90d7-aa57ca9b3658)

**Comment by wan-x-ai** [comment]
Yes, we dynamically adjust flow_shift based on the resolution of the training videos. For implementation details,  please refer to the flux module/documentation.
